test/kubernetes/ftlib-test.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```

Comment:
Are we also aiming to validate that this works with Travis + Minikube?

Reply:
No, I don't think so.